chain/gen/genesis/miners.go

```diff
@@ -584,14 +587,23 @@ func currentEpochBlockReward(ctx context.Context, vm *vm.VM, maddr address.Addre
 	}
 
 	// TODO: This hack should move to reward actor wrapper
```

Comment:
Does this TODO still represent our current plans?  If so I can look into scoping this work on the actors side.  However it seems like our idea of this has changed and this is no longer considered a hack.  If so let's remove the TODO.

Reply:
Ideally this would move into the reward actor _wrapper_. That is, the shim in chain/actors/builtin/reward. We have some abstractions there around message sending.